bundle/verify.go

```diff
@@ -20,9 +20,45 @@ import (
 	"github.com/pkg/errors"
 )
 
+const defaultVerifierID = "_default"
+
+var verifiers map[string]Verifier
+
+// Verifier is the interface expected for implementations that verify bundle signatures.
+type Verifier interface {
+	VerifyBundleSignature(SignaturesConfig, *VerificationConfig) (map[string]FileInfo, error)
+}
+
+// VerifyBundleSignature will retrieve the Verifier implementation based
+// on the Plugin specified in SignaturesConfig, and call its implementation
+// of VerifyBundleSignature. VerifyBundleSignature verifies the bundle signature
+// using the given public keys or secret. If a signature is verified, it keeps
+// track of the files specified in the JWT payload
+func VerifyBundleSignature(sc SignaturesConfig, bvc *VerificationConfig) (map[string]FileInfo, error) {
+	// default implementation does not return a nil for map, so don't
+	// do it here either
```

Comment:
💭 A bit of an odd choice, isn't it? With the default implementation, I mean.

Reply:
🤷   I'm just following the [original implementation](https://github.com/godaddy/opa/blob/458d87489bc09dd4df6ff06ca8e01439ba8370c1/bundle/verify.go#L25)

Reply:
Yeah, it's not a big deal. Thanks!